cyberdrop_dl/crawlers/luxuretv.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from cyberdrop_dl.crawlers.crawler import Crawler, SupportedPaths
+from cyberdrop_dl.data_structures.url_objects import AbsoluteHttpURL
+from cyberdrop_dl.utils import css
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+class Selector:
+    VIDEO_PLAYER = "video.video-js > source"
+    TITLE = "h1.title-right"
+    NEXT_PAGE = "div#pagination > a:-soup-contains('Suivante »')"
```

Comment:
They also have english and spanish versions of the site: en.luxuretv, es.luxuretv

I think changing it to just this will still work:
```suggestion
    NEXT_PAGE = "div#pagination > a:-soup-contains('»')"
```